frontend/src/Components/Form/HintedSelectInputOption.js

```diff
@@ -18,38 +18,46 @@ function HintedSelectInputOption(props) {
   } = props;
 
   return (
-    <EnhancedSelectInputOption
-      id={id}
-      depth={depth}
-      isSelected={isSelected}
-      isDisabled={isDisabled}
-      isHidden={isDisabled}
-      isMultiSelect={isMultiSelect}
-      isMobile={isMobile}
-      {...otherProps}
-    >
-      <div className={classNames(
-        styles.optionText,
-        isMobile && styles.isMobile
-      )}
+    <div>
+      <EnhancedSelectInputOption
+        id={id}
+        depth={depth}
+        isSelected={isSelected}
+        isDisabled={isDisabled}
+        isHidden={isDisabled}
+        isMultiSelect={isMultiSelect}
+        isMobile={isMobile}
+        {...otherProps}
       >
-        <div>{value}</div>
+        <div className={classNames(
+          styles.optionText,
+          isMobile && styles.isMobile
+        )}
+        >
+          <div>{value}</div>
 
-        {
-          hint != null &&
-            <div className={styles.hintText}>
-              {hint}
-            </div>
-        }
-      </div>
-    </EnhancedSelectInputOption>
+          {
+            hint != null &&
+              <div className={styles.hintText}>
+                {hint}
+              </div>
+          }
+        </div>
+      </EnhancedSelectInputOption>
+      {
+        ((props.name === 'language' || props.name === 'movieInfoLanguage') && id === props.minId) &&
```

Comment:
Seems a bit hacky, I think we are better off making a property here in `HintedSelectInputOption` like `isDivider` or something. Then create a new component/input type for the `LanguageSelectInput` where this is handled... It would be similar to how IndexerFlagSelectInput is handled. 

This way we don't have to update this case by case logic every time we want to use the divider in a select.

Reply:
Yeah, It felt a little hacky writing it up. I'll see what I can do with this. Thanks